src/Kernel/Object.class.st

```diff
@@ -109,6 +109,14 @@ Object class >> howToModifyPrimitives [
 	self error: 'comment only'
 ]
 
+{ #category : 'initialization' }
+Object class >> initialize [
+
+	Warning signal:
+		'Proceeding will call `Behavior>>initialize` and this method will update my superclass and my method dictionary. Are you sure you want to proceed?'.
+	^ super initialize
```

Comment:
Well the point here would be to not do the super initialize at all @jecisc otherwise this breaks everything.

I'm against resumable exceptions just for the sake of it. If we can avoid them, I vouch for that.